poly-commit/src/linear_codes/mod.rs

```diff
@@ -409,9 +430,9 @@ where
             ));
         }
         let leaf_hash_params: &<<C as Config>::LeafHash as CRHScheme>::Parameters =
-            vk.leaf_hash_params();
+            vk.leaf_hash_param();
```

Comment:
if you are renaming it, then let's rename everywhere, e.g. here this `let leaf_hash_params`

Reply:
Good point! Fixed.